rest/accounts/v1/credentials_aws.go

```diff
@@ -152,6 +161,81 @@ func (c *ApiService) ListCredentialAws(params *ListCredentialAwsParams) (*ListCr
 	return ps, err
 }
 
+//Lists CredentialAws records from the API as a list. Unlike stream, this operation is eager and loads 'limit' records into memory before returning.
+func (c *ApiService) ListCredentialAws(params *ListCredentialAwsParams, limit *int) ([]*ListCredentialAwsResponse, error) {
+	params.SetPageSize(client.ReadLimits(params.PageSize, limit))
+
+	response, err := c.PageCredentialAws(params, "", "")
+	if err != nil {
+		return nil, err
+	}
+
+	curRecord := 0
+	var records []*ListCredentialAwsResponse
+
+	for response != nil {
+		records = append(records, response)
+
+		var record interface{}
+		if record, err = client.GetNext(response, &curRecord, limit, c.getNextListCredentialAwsResponse); record == nil || err != nil {
+			return records, err
+		}
+
+		response = record.(*ListCredentialAwsResponse)
+	}
+
+	return records, err
+}
+
+//Streams CredentialAws records from the API as a channel stream. This operation lazily loads records as efficiently as possible until the limit is reached.
+func (c *ApiService) StreamCredentialAws(params *ListCredentialAwsParams, limit *int) (chan *ListCredentialAwsResponse, error) {
```

Comment:
can the `limit` be added to the `ListCredentialAwsParams`?

Reply:
That doesn't seem right to me since `params` hold the form params and limit isn't one and besides, that would be complicated to implement in the oai-generator.